ansible/roles/host-ocp4-installer/tasks/main.yml

```diff
@@ -4,26 +4,35 @@
 - name: Generate install_config.yaml
   import_tasks: generate_install_config.yml
 
+- name: Run openshift-install create manifests
```

Comment:
Shouldn't we have a conditional here? If we're not doing either of the workarounds, we don't need to do this.

Reply:
Yeah, but it hardly matters. Not running this step complicates the code for very little difference in execution time.